comp110-worksheet-E-master/20176.0.txt

```diff
@@ -0,0 +1,39 @@
+# move in.a into the acc and move in.b to the cell to the right subtacts in.b from in.a and move's it down
+# then negateds the acc and moves it down it doesn't go into the worng output as @8 altinates it so in.a go to
+# out.a and in.b go to out.b
```

Comment:
Comments would be easier to follow if they were written throughout the code to which they refer, rather than in a block at the top.